tools/configwrappers/wrapper.go

```diff
@@ -0,0 +1,71 @@
+package configwrappers
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/smartcontractkit/ccip-owner-contracts/tools/gethwrappers"
+)
+
+type WrappedManyChainMultisig struct {
+	gethwrappers.ManyChainMultiSig
+}
+
+type ContractDeployBackend interface {
+	bind.ContractBackend
+	bind.DeployBackend
+}
+
+func DeployAndConfigureManyChainMultisig(auth *bind.TransactOpts, backend ContractDeployBackend, config *Config) (common.Address, []*types.Transaction, *WrappedManyChainMultisig, error) {
```

Comment:
I could see the MCMS Config struct potentially being helpful, but just a Config.ToArgs() function would be sufficient for use at a higher level. The thin wrapper methods on a WrappedManyChainMultisig functions don't provide much value

Reply:
Doesn't it become additional and unnecessary cognitive load to know that during SetRoot, signatures need to be sorted by their dervied address in increasing order. Because thats a requirement for SetRoot calls to succeed

Reply:
Hm ok yeah thats fair